src/output/console.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace GOL\Output;
+
+use GOL\Boards\Board;
+use Ulrichsg\Getopt;
+
+/**
+ * Prints the board to the console
+ *
+ * Use write() to write the Board data to the buffer.
+ * and flush() to print the buffer.
+ */
+class Console extends Output
+{
+    private $buffer = [];
+
+    /**
+     * Writes the current board to the Output.
+     *
+     * @param Board $_board Board to output.
+     */
+    public function write(Board $_board): void
+    {
+        $this->buffer[] = $_board->getGrid();
+    }
+
+    /**
+     * Writes the data to the console.
+     */
+    public function flush(): void

```

Comment:
Do I understand that correctly that the output is only printed to the console once all generations are calculated?
I would have found it better to output it directly in the `write()`-method and do nothing in the `flush()`-method. (At least for the console output)